### DISCUSSION TITLE
How do you make it spawn the room templates with its predefined Layer and Tag?

### DISCUSSION THREAD

**Discussion by AiRbU380** [OndrejNepozitek/Edgar-Unity#130] [Q&A] [answered]
This might be a dumb question but this is my first project with unity and c sharp so...

**Comment by OndrejNepozitek** [comment, marked as answer]
Hello, please check this page in the docs https://ondrejnepozitek.github.io/Edgar-Unity/docs/other/faq/#changes-to-a-room-template-are-lost-after-a-level-is-generated

**Reply by AiRbU380** [reply]
thank you
